tests/PhpSpreadsheetTests/Style/FontTest.php

```diff
@@ -39,4 +39,25 @@ public function testSuperSubScript(): void
         self::assertTrue($font->getSuperscript());
         self::assertFalse($font->getSubscript(), 'False remains unchanged');
     }
+
+    public function testSize(): void
+    {
+        $spreadsheet = new Spreadsheet();
+        $sheet = $spreadsheet->getActiveSheet();
+        $cell = $sheet->getCell('A1');
+        $cell->setValue('Cell A1');
+        $font = $cell->getStyle()->getFont();
+
+        self::assertEquals(11, $font->getSize(), 'The default is 11');
+
+        $font->setSize(12);
+        self::assertEquals(12, $font->getSize(), 'Accepted new font size');
+
+        $font->setSize('');
+        self::assertEquals(10, $font->getSize(), 'Set to 10 after trying to set an invalid value.');
+        
+        $font->setSize(12);
+        $font->setSize(0);
+        self::assertEquals(10, $font->getSize(), 'Set to 10 after trying to set an invalid value.');
+    }
```

Comment:
Tests should really include setting for negative numeric values, and for null or non-empty, non-numeric string